alica_tests/src/test/Expr/src/Behaviour/ReadConfigurationBehaviour.cpp

```diff
@@ -34,7 +34,8 @@ void ReadConfigurationBehaviour::run(void* msg)
 void ReadConfigurationBehaviour::initialiseParameters()
 {
     /*PROTECTED REGION ID(initialiseParameters1588061129360) ENABLED START*/
-    getParameter("TestValue", testValue);
+    // getPlanContext()->getParameter("TestValue", testValue);
+    testValue = "1";
```

Comment:
can't we get rid of this behaviour since we no longer have configurations

Reply:
Configuration cleanup will follow in a later cleanup pr.